chimg/chroot.py

```diff
@@ -134,8 +134,32 @@ def _snaps_install(self):
         # install snapd only if not already explicitly installed
         if "snapd" not in snap_infos.keys():
             self._snap_install("snapd", "stable")
+
+        # write seed.yaml
+        self._snaps_create_seed_yaml(snap_infos)
         logger.info("Snaps installed")
 
+    def _snaps_create_seed_yaml(self, snap_infos: Dict[str, SnapInfo]):
+        """
+        Write out the seed.yaml file based on the given snap
+        """
+        Path(f"{self._ctx.chroot_path}/var/lib/snapd/seed/").mkdir(parents=True, exist_ok=True)
+        seed_yaml = f"{self._ctx.chroot_path}/var/lib/snapd/seed/seed.yaml"
```

Comment:
This is used twice, you can make it a variable `f"{self._ctx.chroot_path}/var/lib/snapd/seed/"`

Reply:
I'll extract paths in another commit given that this is used all over the place.

Reply:
done now